docs/guides/admin/email/mailgun.md

```diff
@@ -1,9 +1,11 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-## introduction
+# Email Setup - Using Mailgun
+
+## Introduction
 SeAT requires email to be setup to allow for things like password reminders and notifications to be sent. This guide will attempt to describe how to go about setting up your email using the [mailgun](https://www.mailgun.com/) service. Though mailgun is a commercial service, you get to send 10k emails for free per month. It also provides epic stats for you to track emails with etc.
```

Comment:
see my other comment: passwords won't be sended out anymore.
Generally i'd say Mail-Setup is now optional